client/sites-dashboard/hooks/use-site-status.ts

```diff
@@ -0,0 +1,34 @@
+import { useI18n } from '@wordpress/react-i18n';
+import { SiteExcerptData } from 'calypso/data/sites/site-excerpt-types';
+import type { SiteStatus } from '@automattic/components/dist/types/sites-table/use-sites-table-filtering';
```

Comment:
Don't like doing that, but I couldn't find a way to import this type from the package. The index file is a JS one, so I couldn't export the types. Hope it's good enough and, if it isn't, I'm looking for alternatives.

Reply:
One way would be to rename `components/src/index.js` to `index.ts`. But doing that might require some other tidy-up of the components package. I'm not sure how trivial that will be.

I also tried creating an `index.ts` file to sit beside the `index.js` to see whether it'd merge them together in the build. It didn't seem to work though.